sanic/sanic.py

```diff
@@ -275,8 +274,8 @@ def converted_response_type(self, response):
 
     def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
             after_start=None, before_stop=None, after_stop=None, ssl=None,
-            sock=None, workers=1, loop=None, protocol=HttpProtocol,
-            backlog=100, stop_event=None, register_sys_signals=True):
+            sock=None, workers=1, protocol=HttpProtocol, backlog=100,
```

Comment:
Do you think it would be smart to leave the `loop` param (with a deprecation notice from `warnings.warn`) in for `0.3.0` and then officially remove it in the next release?